dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -221,3 +221,11 @@ LDAP_REGISTRY_CUSTOM_MAPPER_FAILED.useraction=If the exception was unexpected, r
 LDAP_REGISTRY_INVALID_MAPPING=CWIML4504W: The custom X.509 certificate mapper returned an empty or null value. 
 LDAP_REGISTRY_INVALID_MAPPING.explanation=The custom X.509 certificate mapper must return a non-empty and non-null value or throw one of the named exceptions.
 LDAP_REGISTRY_INVALID_MAPPING.useraction=Modify the custom X.509 certificate mapper implementation so that it does not return an empty or null value. If a valid value cannot be returned, the implementation must throw a checked exception. 
+
+LOGINPROPERTY_OVERRIDE_USERFILTER=CWIML4505W: The defined userFilter attribute will be ignored since there are loginProperty attributes defined. The user filter will be dynamically generated.
```

Comment:
Suggested update

>The defined userFilter attribute is ignored because loginProperty attributes are defined. The user filter is dynamically generated.